frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -30,7 +30,7 @@
 	split_emails,
 )
 
-MAX_RETRY_COUNT = 3
+MAX_RETRY_COUNT = cint(frappe.db.get_system_setting("email_retry_count")) or 3
```

Comment:
Semgrep is dead rn else it would've caught it. 

This needs to be moved in function call to support multitenancy. (Global variables are evaluated just once)